documentation/RLC-quickstart.md

```diff
@@ -1,71 +1,96 @@
+
+1. **Project Folder Structure**: the typical structure is `sdk/{servicename}/{servicename}-{modulename}-rest`, e.g., `sdk/agrifood/agrifood-farming-rest`. That folder is under {SDK_REPO_ROOT} and will be your **${PROJECT_ROOT} folder**. 
+
+1. **Package Name Convention**: follow the format `@azure-rest/{service-name}-{module}`, like `@azure-rest/agrifood-farming`.
+
+# Steps to generate RLC
+
+1. **Configure tspconfig.yaml in spec repository** 
    
-   In TypeSpec project, modify `package.json` to add dependency for TypeScript emitter, then run `npm install` again to install `@azure-tools/typespec-ts`.
+   In your specs repository, update or create `tspconfig.yaml` to configure the TypeScript emitter. Replace `YOUR_SERVICE_DIRECTORY`, `YOUR_SERVICE_FOLDER` and `YOUR_PACKAGE_NAME` with your specific details.
+
+   ```yaml
+   parameters:
+     "service-dir":
+       default: "YOUR_SERVICE_DIRECTORY"
+
+   emit: ["@azure-tools/typespec-ts"]
+
+   options:
+     "@azure-tools/typespec-ts":
+       package-dir: "YOUR_SERVICE_FOLDER-rest"
+       packageDetails:
+         name: YOUR_PACKAGE_NAME
+         description: "SHORT_DESCRIPTION"
+         version: "1.0.0-beta.1"
+   ```
+
 
-   ```json
-   "dependencies": {
-     ...
-     "@azure-tools/typespec-ts": "latest"
-   },
+2. **Configure your tsp-location.yaml**  
+   
+   You can update `tsp-location.yaml` under project root folder to set the typespec project. You can refer to the [tsp-location.yaml](https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/TypeSpec-Project-Scripts.md#tsp-locationyaml) which describes the supported properties in the file.
+   
+   If this is the first time to generate code you need to create the project folder.
+   ```shell
+   mkdir -p sdk/agrifood/agrifood-farming-rest # create project folder
+   cd sdk/agrifood/agrifood-farming-rest # enter project folder
    ```
+   Then prepare the `tsp-location.yaml` and here is an example:
 
-2. **Configure TypeScript emitter in tspconfig.yaml**
+   ```yaml
+   directory: specification/agrifood/DataPlane
+   commit: b646a42aa3b7a0ce488d05f1724827ea41d12cf1 # the commit id you'd like to refer for generation
+   repo: Azure/azure-rest-api-specs
+   ```
 
-    In TypeSpec project, modify (or create) `tspconfig.yaml` and configure the SDK generated, using the emitter options on `@azure-tools/typespec-ts`
-    
-    ```yaml
-    emit:
-      - "@azure-tools/typespec-ts"
-    options:
-      "@azure-tools/typespec-ts":
-        title: Farmbeats
-        generateMetadata: true
-        generateTest: true
-        "emitter-output-dir": "{output-dir}"
-        packageDetails:
-          name: "@azure-rest/agrifood-farming"
-          description: "Farmbeats Client"
-          version: "1.0.0-beta.1"
+3. **Generate code**
 
-    ```
+    Run the following two scripts from project directory (i.e sdk/agrifood/agrifood-farming-rest) to generate the code:
```

Comment:
Also use the new tool instead of the PS1 script

Reply:
updated